### PR TITLE
Extra helpers + oversight fixes

### PR DESCRIPTION
**Pull request by rymiel**
bulmatoast helper, fetch POST helper, also uses the websocket send extension where it was missed

Also resolves headers being sent as a kotlin map instead of a js object

### CODE REVIEW

File: src/jsMain/kotlin/net/kyori/adventure/webui/js/ExternalFunctions.kt [hunk 1]
```diff
@@ -17,3 +18,6 @@ public external class bulmaToast {
         public fun setDefaults(settings: Json)
     }
 }
+public fun bulmaToast.Companion.toast(message: String, type: String) {
```
Comment (kezz) [inline src/jsMain/kotlin/net/kyori/adventure/webui/js/ExternalFunctions.kt:21]: Should have a newline here. Also, let's make `type` default to `"is-success"`.

File: src/jsMain/kotlin/net/kyori/adventure/webui/js/Main.kt [hunk 2]
```diff
@@ -581,13 +550,10 @@ private fun checkClickEvents(target: EventTarget?, typesToCheck: Collection<Even
...
                 bulmaToast.toast(
-                    json(
-                        "message" to
-                            "<p><b>Click Event</b></p><p>Action: <i>${
-                            clickAction.replace('_', ' ').replaceFirstChar(Char::uppercase)
-                            }</i></p><p>Content: <i>$content</i></p>",
-                        "type" to "is-info"
-                    )
+                    "<p><b>Click Event</b></p><p>Action: <i>${
+                    clickAction.replace('_', ' ').replaceFirstChar(Char::uppercase)
```
Comment (kezz) [inline src/jsMain/kotlin/net/kyori/adventure/webui/js/Main.kt:554]: this is so ugly.. not sure if there's anything we could do here - perhaps a text block? or an extension function or something?

**Review by kezz — CHANGES_REQUESTED** [review]
Minor comments - lovely PR though! Bit of tidying is fabbbb

File: src/jsMain/kotlin/net/kyori/adventure/webui/js/ExternalFunctions.kt (hunk 1 above)
Comment (rymiel) [reply src/jsMain/kotlin/net/kyori/adventure/webui/js/ExternalFunctions.kt:21]: Okay! I was unsure if `is-success` or `is-info` should be default

**Review by kezz — APPROVED** [review]
Beautiful! Thank you for the PR and apologies for taking so long to review this.

**Pull request merged**